src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -723,4 +723,9 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Property SourceProvider is null.</value>
     <comment>Don't localize 'SourceProvider'</comment>
   </data>
+  <data name="Error_AttemptingToPackSDKproject" xml:space="preserve">
+    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` to pack this project instead. You may revert to the legacy pack behavior by setting the  '{0}' environment variable to 'true'.</value>
```

Comment:
Hey @JonDouglas 
I'd be appreciate a quick review of this. Since last you looked at it, we've added the revert instructions. 

It should be good, but lmk if you have any feedback.

Reply:
> You may revert to the legacy pack behavior by setting the  '{0}' environment variable to 'true'.

- >

> You can opt-out of this behavior by setting the  '{0}' environment variable to 'true'.

Food for thought. legacy pack sounds weird. It never worked in the first place, is it really legacy?

Reply:
The sentence is actually updated and GH just shows it as `Outdated`. 

Current version is:

> The pack command for SDK-style projects is not supported, use dotnet pack to pack this project instead. To force nuget to run pack command, you can set the '{0}' environment variable to 'true' but this is not recommended.